external/harvestPackages/harvestPackages.props

```diff
@@ -67,6 +67,9 @@
 <PackageReference Include="System.Json">
   <Version>4.4.0</Version>
 </PackageReference>
+<PackageReference Include="System.Memory">
```

Comment:
I assume you no longer need to harvest this package correct?

Reply:
It's still needed as harvestPackages.depproj just deals with downloading the packages into the packages folder and manually updating the versions in the harvestPackages.props. It doesn't do anything related to harvesting but it is mostly used for harvesting. I suggest we rename `harvestPackages` to `downloadPackages`.

Reply:
Oh right, it gets also restored inside binplacePackages.depproj. Then it makes sense to remove it from harvestPackages.